src/components/accordionPillSelect/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import PillSelect from '../pillSelect';
+import { SelectProps } from '../coreSelect';
+import PlusIcon from '../../images/plus.inline.svg';
+import MinusIcon from '../../images/minus.inline.svg';
+
+function AccordionPillSelect<T>({ title, ...props }: SelectProps<T>) {
```

Comment:
Should ...props -> be changed to named props?

Reply:
Not really since the same props are just given to the inner component `<PillSelect />`